.editorconfig

```diff
@@ -0,0 +1,11 @@
+
+root = true
+
+[*]
+indent_style = true
+end_of_line = lf
```

Comment:
@madmaxoft Can you override `.editorconfig` to use your own preferred settings on your editor? If you can do that then we can set `end_of_line=lf`. The editorconfig should only be a recommendation, with the final control up to the user